src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -0,0 +1,318 @@
+section.
+
+#### `syncDrafts`
+
+Used to sync a list of [JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk)
+[InventoryEntryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntryDraft.java)
+objects.
+
+````java
+// instantiating an inventory sync
+final InventorySync inventorySync = new InventorySync(inventorySyncOptions);
+
+// execute the sync on your list of inventory entries
+inventorySync.syncDrafts(inventoryEntriesDrafts);
+````
+
+**Important!**
+Before using `syncDrafts` you should assert that every [InventoryEntryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntryDraft.java)
+from input list which contains [Reference](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java)
+to a supply channel has either that reference expanded, or has that reference not expanded but instead has supply channel
+key provided in place of reference `id` (that means calling [getId()](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java#L26)
+function on a reference instance the `String` that represents supply channel key would be returned). By expanded reference
+we mean that calling [getObj()](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/Reference.java#L52)
+function on a reference's instance a [Channel](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/channels/Channel.jav)
+object that contains its key would be returned. More information about why it is important can be found in
+[FAQ](#why-is-it-important-to-provide-extended-supply-channel-references-in-entries-from-input-lists-of-sync-process)
+section.
+
+#### `getStatistics`
+
+Used to get an object  containing all the stats of the sync process; which includes a report message, the total number
+of updated, created, failed, processed inventory entries and the processing time of the sync in different time units and in a
+human readable format.
+````java
+inventorySync.syncDrafts(inventoryDrafts);
+inventorySync.getStatistics().getCreated(); // 1000
+inventorySync.getStatistics().getFailed(); // 5
+inventorySync.getStatistics().getUpdated(); // 995
+inventorySync.getStatistics().getProcessed(); // 2000
+inventorySync.getStatistics().getReportMessage();
+/*
+ * Summary of inventory synchronisation:
+ * 2000 inventory entries were processed in total (1000 created, 995 updated and 5 failed to sync)."
+ */
+ ````
+
+### How to use `InventorySyncUtils` and `InventoryUpdateActionUtils`
```

Comment:
As Roman suggests, we can just get rid of all these code snippets here and just mention that through the InventorySyncUtils and the InventoryUpdateActionUtils the sync can be used to calculate any of the [inventory entries' update actions](http://dev.commercetools.com/http-api-projects-inventory.html#update-an-inventoryentry) individually between an old inventoryEntry and a new one. And provide a link to the IT where they are actually used in code. 

This is to make the README not soo huge.